legal-api/report-templates/template-parts/notice-of-articles/nameTranslations.html

```diff
@@ -0,0 +1,8 @@
+{% if nameTranslations|length > 0 %}
```

Comment:
Does this work if nameTranslations is not defined?

Reply:
Yes